backend/src/pilcom_cli/mod.rs

```diff
@@ -0,0 +1,93 @@
+mod json_exporter;
+
+use crate::{Backend, BackendBuilder, Proof};
+use ast::analyzed::Analyzed;
+use json::JsonValue;
+use number::{write_polys_file, DegreeType, FieldElement};
+use std::{
+    fs,
+    io::{self, BufWriter},
+    path::Path,
+};
+
+pub struct PilcomCliBuilder;
```

Comment:
pilcom is just the compiler, maybe name it pilstark?

Reply:
I think the correct name here is indeed pilcom, because we are writing an output in the specific format of the tool. Imagine pilcom starts supporting other proof systems besides eSTARK, our pilcom output should still work.